packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -85,10 +85,8 @@ export default function ConvertToLinksModal( { onClose, clientId } ) {
 				{
 					per_page: MAX_PAGE_COUNT,
 					_fields: PAGE_FIELDS,
-					// TODO: When https://core.trac.wordpress.org/ticket/39037 REST API support for multiple orderby
-					// values is resolved, update 'orderby' to [ 'menu_order', 'post_title' ] to provide a consistent
-					// sort.
-					orderby: 'menu_order',
+					//TODO: orderby menu_order,title when https://core.trac.wordpress.org/ticket/39037 is implemented
+					orderby: 'title',
```

Comment:
This feels more disruptive when we have ordered pages than what's in trunk, because it makes all the links jump around, whereas before they would only change if you had more than one item with the same order. If pages aren't ordered though, it behaves better. Not sure which is best 😕 I wonder how commonly the "order" feature is used?

Alternatively, would passing `'menu_order, ID'` instead of `'menu_order, post_title'` to `get_pages` solve the inconsistency issue? That would render the page list exactly the same as this query does with `menu_order`. The downside is that it won't give us the same order as the classic Pages widget with the menu order option selected, so if the main concern is being able to use Page List as a replacement for the widget, we should stick to `'menu_order, post_title'`.

#31382 doesn't mention the Navigation at all, so I assume the problem is mostly with the order within Page List itself, which kind of points to it being used as a replacement widget.

Reply:
The rest api doesn't support multiple sorts, so results will be inconsistent depending on query size. We could leave it as menu_order. 

For multiple sorts, I do have https://github.com/WordPress/wordpress-develop/pull/1186 but I got a little stuck since it has a breaking change for an existing hook

Reply:
We can always revisit this later once the REST issue is fixed. `title` will work better with no ordering, and that's the default state, so I guess we can go with that for now 😄